js/web/lib/wasm/binding/index.ts

```diff
@@ -0,0 +1,39 @@
+import wasmModuleFactory, {BackendWasmModule} from './onnxruntime_wasm';
```

Comment:
multi-threading support is not in this PR?

Reply:
no, but I am working on with it